chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -507,7 +507,7 @@ async def test_duplicate_output() -> None:
 async def test_block_cost_exceeds_max() -> None:
     mempool_manager = await instantiate_mempool_manager(zero_calls_get_coin_records)
     conditions = []
-    for i in range(2400):
+    for i in range(3800):
```

Comment:
Correct

Reply:
Hmm that doesn't seem to be correct.

With the inclusion of byte cost in https://github.com/Chia-Network/chia_rs/pull/679 we now have a *higher* cost than what's currently in main, not lower.

This highlighted line for example goes down from needing `2400` items in main, to needing just `2389` (not `3800`), due to cost becoming higher, and the expected cost for https://github.com/Chia-Network/chia-blockchain/pull/18213/files#diff-6a5a9dba08be65c628eff46226c4f7d83c6dbf4bcb9c15649c228b9d8b244c76R581 as another example goes up from main's `10268283` to `19144088`, not down to `6600088`.

Reply:
> With the inclusion of byte cost in Chia-Network/chia_rs#679 we now have a higher cost than what's currently in main, not lower.

I suspect this is a bug in the rust implementation. Although, as it turns out, the true reason these numbers had to change (so much) is because of the current issue with the rust implementation (not including size cost)

Reply:
Sent a fix for that in https://github.com/Chia-Network/chia_rs/pull/680 and adapted to the fix here with https://github.com/Chia-Network/chia-blockchain/pull/18537.